buildscripts/semantic-convention/generate.sh

```diff
@@ -1,30 +1,32 @@
 #!/bin/bash
 
-DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+ROOT_DIR="${SCRIPT_DIR}/../../"
+# freeze the spec version to make SemanticAttributes generation reproducible
+SPEC_VERSION=d328331ce9dc6dec871b647b4e37a84b5396516e
```

Comment:
can we also reference the *version* of the spec that we're referencing, so it's clear we're not pulling in arbitrary SHAs?

Reply:
But we are pulling arbitrary SHAs 😅 

We can use tags here instead of hashes, e.g. `v0.7.0` but I'm afraid that just using tags will be very limiting. Spec `v0.7.0` is pretty recent, but `v0.6.0` was released 5 months ago.

Reply:
We could maybe add a comment with the `git describe` of that version next to it. (Or can we use `git describe` result directly?)

Reply:
In this case it would be `v0.7.0-15-gd328331`. And git seems to be able to parse this as revision:
```
$git log v0.7.0-15-gd328331
commit d328331ce9dc6dec871b647b4e37a84b5396516e
Author: Christian Neumüller <christian.neumueller@dynatrace.com>
Date:   Thu Nov 26 13:15:29 2020 +0100

    Add .gitattributes & normalize line endings (#1230)

    Co-authored-by: Armin Ruech <armin.ruech@dynatrace.com>

commit 2aa62dfe1efff9d25d935c52d579c44418c925d8
Author: William Armiros <54150514+willarmiros@users.noreply.github.com>
Date:   Tue Nov 24 20:17:42 2020 -0600

    Add semantic conventions for cloud provider-specific resource attributes (#1099)
```

Reply:
Nice! That looks great, much better than sha1.

Reply:
Unfortunately it doesn't work

```
fatal: couldn't find remote ref v0.7.0-15-gd328331
```

Reply:
I fine using the SHA, but would like to just have a comment about how we got it (i.e. from a release tag, etc).